app/Http/Controllers/Item/ItemController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\Item\ItemRepository;
+use App\Item;
+use app\Http\Requests\CreateItemRequest;
+
+class ItemController extends Controller
+{
+    private $itemRepository;
+
+    public function __construct(ItemRepository $itemRepository)
+    {
+        $this->itemRepository = $itemRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // TO-DO
+        // Ask fro help from Jeseph of how to use the config values on pagination
+        // get the pagination number or a default
+        $items = $this->$itemRepository ?? 10;
+        return view('items' . config('settings.pagination.small') . '.index')->with(['$items' => $items]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('items.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  CreateItemRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CreateItemRequest $request)
+    {
+        $data = $request->all();
+        $data['name'] = strtoupper($data['size'] . "' " . $data['code'] . ' ' . $data['brand']);
+        Item::create($data);
+        return redirect()->to('items')->withSuccess('Item created Successfully');
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $items
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Item $items)
+    {
+        //return view with Item list
+        return view('items.show', compact('items'));
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit(Item $item)
+    {
+        return view('items.edit')->with(['item' => $item]);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  CreateItemRequest  $request
+     * @param  int  $item
+     * @return \Illuminate\Http\Response
+     */
+    public function update(CreateItemRequest $request, Item $item)
```

Comment:
No `return` statement here. You must have forgotten that you need to redirect the user somewhere after updating the item.